### PR TITLE
RD-3197 Exposed `ProductVersion` as standalone component that can be used outside of `HeaderBanner` context

### PR DESCRIPTION
**Pull request by kubama**
## Description
Exposed `ProductVersion` as standalone component that can be used outside of `HeaderBanner` context

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [X] I verified if that change requires a documentation update.
- [X] I added proper labels to this PR.

## Tests
Updated snapshot and added loki image.

 

## Documentation
Added JSDocs and FS story

### CODE REVIEW

**Comment by qooban** [comment]
I didn't manage to review this PR today. Will provide review tomorrow morning.

File: src/components/layout/ProductVersion/ProductVersion.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+/**
+ * Formats and displays short version label (major.minor) based on passed full Cloudify version string.
+ */
+export default function ProductVersion({ version, style }: ProductVersionProps) {
+    const versionMatches = version.match(/^(\d+)\.(\d+).*$/);
+    const major = !!versionMatches && _.size(versionMatches) >= 2 ? versionMatches[1] : '';
+    const minor = !!versionMatches && _.size(versionMatches) >= 3 ? versionMatches[2] : '';
+    const shortVersion = `${major}${minor ? `.${minor}` : ''}`;
```
Comment (qooban) [inline src/components/layout/ProductVersion/ProductVersion.tsx:23]: Optional:

Probably we can optimize it a bit to have something like:
```suggestion
    const shortVersion = React.memo(() => {
        const versionMatches = version.match(/^(\d+)\.(\d+).*$/);
        const major = !!versionMatches && _.size(versionMatches) >= 2 ? versionMatches[1] : '';
        const minor = !!versionMatches && _.size(versionMatches) >= 3 ? versionMatches[2] : '';
        return `${major}${minor ? `.${minor}` : ''}`;
    }, [version]);
```
NOTE: I didn't check that code. It's more like to show the intention to use `React.memo`.

File: src/components/layout/ProductVersion/ProductVersion.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+    if (_.isEmpty(shortVersion)) return null;
+
+    return <span style={{ color: colors.greyNormal, verticalAlign: 'middle', ...style }}> {shortVersion}</span>;
```
Comment (qooban) [inline src/components/layout/ProductVersion/ProductVersion.tsx:27]: Probably having whitespace before the version string should not be here, but I assume we should keep it to maintain compatibility with Composer, right?

**Review by qooban — APPROVED** [review]
Looks good 👍🏻 

Added two minor comments.

File: src/components/layout/ProductVersion/ProductVersion.tsx (hunk 2 above)
Comment (kubama) [reply src/components/layout/ProductVersion/ProductVersion.tsx:27]: Yes, this is my thinking as well

File: src/components/layout/ProductVersion/ProductVersion.tsx (hunk 1 above)
Comment (kubama) [reply src/components/layout/ProductVersion/ProductVersion.tsx]: Done

**Review by qooban — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**